tests/bluetooth/tester/src/audio/btp_bap.c

```diff
@@ -369,12 +369,17 @@ static const struct btp_handler bap_handlers[] = {
 		.expect_len = BTP_HANDLER_LENGTH_VARIABLE,
 		.func = btp_bap_audio_stream_send,
 	},
-#if defined(CONFIG_BT_BAP_BROADCAST_SINK) || defined(CONFIG_BT_BAP_BROADCAST_SINK)
+#if defined(CONFIG_BT_BAP_BROADCAST_SOURCE) || defined(CONFIG_BT_BAP_BROADCAST_SINK)
 	{
 		.opcode = BTP_BAP_BROADCAST_SOURCE_SETUP,
 		.expect_len = BTP_HANDLER_LENGTH_VARIABLE,
 		.func = btp_bap_broadcast_source_setup,
 	},
+	{
+		.opcode = BTP_BAP_BROADCAST_SOURCE_SETUP_V2,
```

Comment:
I see it is missing already in BAP but supported commands should also be updated, we want to use those on autopts side eventually